test/test_multi_configuration.rb

```diff
@@ -0,0 +1,113 @@
+require 'al-test-utils'
+
+class TestMultiConfiguration < Test::Unit::TestCase
+  include AlTestUtils
+
+  def setup
+    super
+    @original_configs = ActiveLdap::Base.configurations.dup
+  end
```

Comment:
Do we need this?

We already have this: https://github.com/activeldap/activeldap/blob/31b5c9b083f050a488eb2ee741f5596258e399fa/test/al-test-utils.rb#L50